mteb/benchmarks/benchmarks.py

```diff
@@ -25,77 +43,91 @@ def __getitem__(self, index):
         return self.tasks[index]
 
 
+def create_benchmark_list() -> list[type[Benchmark]]:
+    benchmark_categories_cls = list(Benchmark.__subclasses__())
```

Comment:
Ugh this `.__subclasses__` thing is a little black magicky.
I think it would probably make sense to just hard code these instead, or even better, a catalogue registry (though I assume this has crossed your mind and ultimately decided not to go with it).
Also I'm wondering if this function is being used anywhere else. Is this supposed to be for ease of use?

Reply:
I actually removed this part in favor of the other approach

Reply:
so i have deleted this segment